docs/extras/project-ideas/index.md

```diff
@@ -0,0 +1,176 @@
+
+Use fuzzy matching on names, or other techniques. Create a [processor](https://github.com/opentrials/processors) that encapsulates your normalisation routines and submit a [pull request](https://github.com/opentrials/processors/pulls) for peer review.
+
+#### Skills
+
+- Intermediate data wrangling
+- Intermediate Python
+- Knowledge of existing dictionaries/lookup services to clean and normalise data against
+
+### Normalise person data
+
+#### What
+
+As with the above idea on organisation data, similar opportunities exist for normalising person data in the OpenTrials database. This data has been extracted into a dedicated table for the Person entity, but needs much cleaning and normalisation in order to turn it into a usable resource.
+
+#### How
+
+As lookup tables for persons are harder to come by than for organisations, the suggested approach would be to do fuzzy matching across the names as a first step to creating a normalised dataset.
+
+Beyond this important step, one could then use the cleaned list of names against a generic search engine, and look for matches with a high degree of alignment to the medical domain, and start to extract additional data on these persons, such as academic publications, association with research departments etc. Additionally, for trials with [linked publications](http://explorer.opentrials.net/search?q=&has_publications=true), names matches could be looked up against the list of authors for publications known to relate to the trial (NB. multiple publications may relate to a particular trial).
+
+Building such a body of knowledge, this could then be triangulated with the rest of the OpenTrials database, identifying new academic publications related to trials, for example.
+
+#### Skills
+
+- Advanced data wrangling
+- Intermediate Python
+
+### Collect new data sources
+
+#### What
+
+In OpenTrials, we have [collectors](https://github.com/opentrials/collectors), which use modules and functions of Python code to get data from some external source into the OpenTrials warehouse database for further processing.
+
+There are many data sources available. So far we've build out the spine of the database with core clinical trial records, but there are tens of additional sources that can potentially be matched onto the clinical trials we know about, and provide value to the database.
+
+#### How
+
+See the [documentation for all existing collectors](https://github.com/opentrials/collectors/tree/master/docs/collectors) and start your own with a new data source! A good place to begin would be to see if any collectors have not yet been written for the 16 [WHO Primary Registries](http://www.who.int/ictrp/network/primary/en/).
```

Comment:
There're 15 primary registries so far (not 16). Maybe we should use "scraping" somewhere, as it's clearer (to who knows what scraping is) what we're talking about.